app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Recipe:<%= @recipe.name %></h1>
+<h3>Complexity: <%= @recipe.complexity %></h3>
+<h3>Genre: <%= @recipe.genre %></h3>
+
+<h1>Ingredients:</h1>
+<% @ingredients.each do |ingredient| %>
```

Comment:
Here is where I would use your `@recipe` from the controller and do a `@recipe.ingredients.each do`.